.gitignore

```diff
@@ -15,5 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-
-.DS_Store
```

Comment:
Shouldn't that be in [your global .gitignore](http://devoh.com/blog/2013/01/global-gitignore)? To keep this .gitignore focused on ember-infinity. And so you don't have to add your personal ignores to every project you work on.